css/design_system/components/card.scss

```diff
@@ -0,0 +1,114 @@
+ /**
+ * @title Basic Card
+ * @category components
+ * @component_type card
+ * @value s-card
+ * @description The basic card allows for a title, subtitle, content, and footer - all of which are optional. The example below shows a card that does use all options.
+ *
+ * @example
+ * <div class="s-card">
+ *  <div class="s-card-basic-header">
+ *    <h2 class="s-card-title">Card Title</h2>
+ *    <h3 class="s-card-subtitle">Card Subtitle</h3>
+ *  </div> 
+ *  <div class="s-card-content">
+ *    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum risus odio quis sit mattis in luctus. Dignissim auctor aliquam suspendisse non in ac, purus. Aliquam arcu justo quam lorem at id proin morbi. A arcu accumsan, eu id fringilla non.</p>
+ *    <p class="s-card-footer">Footer content</p>
+ *  </div>
+ * </div>
+ */
+
+.s-card {
+  @extend .s-p-4;
+  display: inline-block;
+  border-radius: var(--s-space-1);
+  border-top: 8px solid var(--s-color-cyan-600);
+  box-shadow: 0 1px 3px var(--s-color-gray-400);
```

Comment:
Could we use `@extend .s-shadow-sm` here? It's almost the same, probably fine to have that slight color difference.

Reply:
Yes! I wasn't sure how closely to follow the Figma, but this sounds good.